### ISSUE TITLE
NOVAS: Non-convex Optimization via Adaptive Stochastic Search for End-to-End Learning and Control. (arXiv:2006.11992v3 [cs.LG] UPDATED)

### ISSUE THREAD

**Issue by CoffeeKumazaki** [CoffeeKumazaki/arXiv#5510]
NOVAS: Non-convex Optimization via Adaptive Stochastic Search for End-to-End Learning and Control. (arXiv:2006.11992v3 [cs.LG] UPDATED)<br>
https://ift.tt/2StrAHv<br>
<p>In this work we propose the use of adaptive stochastic search as a building block for general, non-convex optimization operations within deep neural network architectures. Specifically, for an objective function located at some layer in the network and parameterized by some network parameters, we employ adaptive stochastic search to perform optimization over its output. This operation is differentiable and does not obstruct the passing of gradients during backpropagation, thus enabling us to incorporate it as a component in end-to-end learning. We study the proposed optimization module's properties and benchmark it against two existing alternatives on a synthetic energy-based structured prediction task, and further showcase its use in stochastic optimal control applications.</p>
<br>
<br>
via eess.SY updates on arXiv.org http://arxiv.org/<br>
